packages/create-qwik/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "create-qwik",
-  "version": "0.102.0",
+  "version": "0.103.0",
```

Comment:
I bumped the version assuming any change to the starters requires this. Looking through old commits I'm not sure if this is needed or desired.